test/famfs_unit.cpp

```diff
@@ -29,13 +29,40 @@ extern "C" {
 #define FAMFS_MPT "/mnt/famfs"
 #define DIRPATH   "/mnt/famfs/testdir42"
 #define TESTFILE  "/mnt/famfs/testdir42/testfile0"
+#define PATH	  256
+#define SYS_UUID_DIR "/opt/famfs"
```

Comment:
Unit test should not test the uuid file "in place" because unit tests should not affect the production configuration of a system.  It should test in /tmp/uuid_test or something like that.